resources/views/head.blade.php

```diff
@@ -17,3 +17,24 @@
         Lang.setLocale("{{ App::getLocale() }}");
     </script>
 @stop
+
+@section('js-localization.head.exported')
+    @if (file_exists(config('js-localization.storage_path') . '/js-localization.min.js'))
```

Comment:
This is quite hard to understand: Why do we check for `${storage_path}/js-localization.min.js` and then set the URL to `url('/vendor/js-localization/js-localization.min.js')` / `url('/js-localization/localization.js')`?

It's not obvious how `${storage_path}/js-localization.min.js` and the URLs relate to each other.

Reply:
You are right of course. If someone would change the `storage_path` config, that would not work indeed. Maybe skip the `@section` altogether? Otherwise one would have to have a function that returns the correlation of `storage_path` and `public_path` (if that exists) to be able to figure out where the exported files actually go. And that might be a little bit out of scope for the plugin?

Reply:
Yeah, that would probably be the easiest solution now: Removing the section again and changing the Readme to tell the user how to use the locale-specific files.

Could still add a section like this in a follow-up PR.